package.json

```diff
@@ -47,7 +47,7 @@
     "copyright": "Copyright © 2022 ${author}",
     "extraMetadata": {
       "//": "--- Prop 'version' will be automatically set by electron-builder ---",
-      "version": "0.0.1"
```

Comment:
🤔

Reply:
Ok yeah I have no idea how this turned up 💀